File: pinyougou/pinyougou-cart-web/src/main/webapp/js/service/cartService.js
app.service("cartService", function ($http) {
    this.getUsername = function () {
        return $http.get("cart/getUsername.do?t=" + Math.random());
    };

    this.findCartList = function () {
        return $http.get("cart/findCartList.do?t=" + Math.random());
    };

    this.addCartToCartList = function (itemId, num) {

        return $http.get("cart/addCartToCartList.do?itemId=" + itemId + "&num=" + num);
    };

});

File: pinyougou/pinyougou-cart/pinyougou-cart-interface/src/main/java/com/pinyougou/cart/service/CartService.java
package com.pinyougou.cart.service;

import com.pinyougou.vo.Cart;

import java.util.List;

public interface CartService {
    /**
     * 增减购物车购买商品数量
     * @param cartList 购物车列表
     * @param itemId 商品sku id
     * @param num 购买数量
     * @return 操作结果
     */
    List<Cart> addCartToCartList(List<Cart> cartList, Long itemId, Integer num);
}
